libtenzir/builtins/formats/zeek_tsv.cpp

```diff
@@ -391,6 +394,34 @@ struct zeek_printer {
       return true;
     }
 
+    auto operator()(view<blob> x) noexcept -> bool {
+      if (x.empty()) {
+        // TODO: Is this actually correct? An empty blob is not unset.
```

Comment:
Zeek has placeholder text for empty and unset values. You want to print the empty value placeholder here instead.

Reply:
Ah, right. I copied this from `string_type`, but it seemed wrong to me. I'll fix this for both types then.